examples/kubernetes/pomerium-certificates.yaml

```diff
@@ -16,8 +16,9 @@ spec:
     - pomerium-authorize.pomerium.svc.cluster.local
     - pomerium-databroker.pomerium.svc.cluster.local
     - pomerium-authenticate.pomerium.svc.cluster.local
-    # TODO - Replace the following entry with your domain space.
-    - "*.localhost.pomerium.io" # Quotes are required to escape the wildcard
+    - authenticate.localhost.pomerium.io
+    # TODO - If you're not using the Pomerium Ingress controller, you may want a wildcard entry as well.
```

Comment:
In some places we call it **the** Pomerium Ingress controller, in others, just Pomerium Ingress Controller.  We also capitalize it differently in different places. 

My preference would be "Pomerium's ingress controller".

Reply:
As far as I understand it, this is because the item in question is a custom... "thing" in Kubernetes and upper-case follows their convention. I found it confusing as well, maybe @travisgroth can provide better context.

Reply:
https://kubernetes.github.io/ingress-nginx/
https://kubernetes.io/docs/concepts/services-networking/ingress-controllers/

use capitalized version

Reply:
I don't think the possessive form fits in very well; other ingress controllers are some version of "X ingress controller".  Would rather agree on a canonical form of "Pomerium Ingress Controller"